c++/WORKSPACE

```diff
@@ -1,61 +1,17 @@
 workspace(name = "capnp-cpp")
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
-
-http_archive(
-    name = "bazel_skylib",
-    sha256 = "f7be3474d42aae265405a592bb7da8e171919d74c16f082a5457840f06054728",
-    urls = [
-        "https://mirror.bazel.build/github.com/bazelbuild/bazel-skylib/releases/download/1.2.1/bazel-skylib-1.2.1.tar.gz",
-        "https://github.com/bazelbuild/bazel-skylib/releases/download/1.2.1/bazel-skylib-1.2.1.tar.gz",
-    ],
-)
-
-load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
-
-bazel_skylib_workspace()
-
-http_archive(
-    name = "brotli",
-    sha256 = "e720a6ca29428b803f4ad165371771f5398faba397edf6778837a18599ea13ff",
-    strip_prefix = "brotli-1.1.0",
-    type = "tgz",
-    urls = ["https://github.com/google/brotli/archive/refs/tags/v1.1.0.tar.gz"],
-)
-
-http_archive(
-    name = "ssl",
-    sha256 = "873ec711658f65192e9c58554ce058d1cfa4e57e13ab5366ee16f76d1c757efc",
-    strip_prefix = "google-boringssl-ed2e74e",
-    type = "tgz",
-    # from master-with-bazel branch
-    urls = ["https://github.com/google/boringssl/tarball/ed2e74e737dc802ed9baad1af62c1514430a70d6"],
-)
-
-# Based on https://github.com/bazelbuild/bazel/blob/master/third_party/zlib/BUILD.
-_zlib_build = """
-cc_library(
-    name = "zlib",
-    srcs = glob(["*.c"]),
-    hdrs = glob(["*.h"]),
-    includes = ["."],
-    # Workaround for zlib warnings and mac compilation. Some issues were resolved in v1.3, but there are still implicit function declarations.
-    copts = [
-        "-w",
-        "-Dverbose=-1",
-    ] + select({
-        "@platforms//os:macos": [ "-Wno-implicit-function-declaration" ],
-        "@platforms//os:linux": [ "-Wno-implicit-function-declaration" ],
-        "//conditions:default": [],
-    }),
-    visibility = ["//visibility:public"],
-)
-"""
-
-http_archive(
-    name = "zlib",
-    build_file_content = _zlib_build,
-    sha256 = "8a9ba2898e1d0d774eca6ba5b4627a11e5588ba85c8851336eb38de4683050a7",
-    strip_prefix = "zlib-1.3",
-    urls = ["https://zlib.net/zlib-1.3.tar.xz"],
+#
+# NOTE: If you're looking at this file to determine how to install Cap'n'Proto in your own
+#       Bazel project, please see the project README; initialization steps here differe from
+#       what they should be in your project.
+#
+
+# Import repositories
+load("//defs:repositories.bzl", "capnp_repositories")
+
+# Only Brotli is actually imported here, because it isn't yet available on Bazel Registry.
+capnp_repositories(
+    omit_skylib = True,
+    omit_boringssl = True,
+    omit_zlib = True,
```

Comment:
The Bazel `WORKSPACE` file for `./c++` now just re-uses the macros that downstream users would be expected to use. This lets the Cap'n'Proto project maintainers focus on their own needs and avoid duplication of code (i.e. because the dependencies used in the project all come from one place).